nrt/data/WECCAdditionalModels/cases.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2023, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+# simulation tool for power systems.
+
+import os
+
+test_cases = []
+standardReturnCode = [0]
+standardReturnCodeType = "ALLOWED"
+forbiddenReturnCodeType = "FORBIDDEN"
+
+###########################################
```

Comment:
The folder for the references should be "reference" and not "references" (for both test cases). Here the nrt don't find the references. Thanks